package day062;

public class example {
    public static void main(String[] args ) throws Exception{

        System.out.println("Hello friends");
//if we have unchecked exceptions try catch will help to catch our exception and
        //will help to continue till the end
      //try {
        //int a=10/0;//unchecked e
          Thread.sleep(2000);//checked exception
//        double arr[]={2.6,8.0};
//        System.out.println(arr[90]);
     //}catch( Exception e){


        System.out.println("no exception");
//try {
//            //Thread.sleep(2000);
//            int []arr={2,5,8,9};
//            System.out.println(arr[10]);
//
//       }catch(ArrayIndexOutOfBoundsException e){
//            //System.out.println(e);
//            System.out.println("you have exception");
//
//       }
//       System.out.println("remainder part ");
//
 }
}
